src/docs/demos/pagination/pagination-demo.html

```diff
@@ -0,0 +1,57 @@
+<div class="mb-4" ng-init="currentPageBasic10 = 1">
+    <h6>Basic (Current Page: {{currentPageBasic10}})</h6>
+    <ngbs-pagination items-per-page="10" total-items="100" on-page-change="currentPageBasic10 = currentPage"></ngbs-pagination>
+</div>
+
+<div class="mb-4">
```

Comment:
Probably not.  Half of them were for testing.  After I made the PR I thought it would be much better to have 1 example with other inputs to let the user adjust all the attributes.  The way the demo page is laid out though, it's very easy for the pagination component to run into the documentation on the right.

Is it OK if that happens if they put in crazy values, or can we somehow avoid it?  z-index maybe, and let it just float over the documentation?